unabridged-works-volume-2/original/10-concerning-persecution.adoc

```diff
@@ -863,6 +937,11 @@ under a pretence of conscience.
 
 [.discourse-part]
 Answer.
+
+[.numbered-group]
+====
+
+[.numbered]
```

Comment:
This is another case like in the last PR, where `Answer.` gets put in its own paragraph by the new formatting, which we don't want.  Let's do the same here, remove the  DP on line 938 and make this just a .numbered, like this:

```ASCIIDOC
[.numbered-group]
====

[.numbered]
_Answer 1._ Conscience is of God;
```